src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -130,3 +137,18 @@ const MerchantInfoContainer = styled.div`
     flex-direction: column-reverse;
   }
 `;
+
+const FlyerContainer = styled.div`
+  margin: 3vw 7vw;
+  font-weight: bold;
+  
+  a {
+    color: #a7182d;
```

Comment:
Are you sure this color isn't defined in a shared var somewhere?

Reply:
I just checked and it's not. Maybe we should?

Reply:
Yeah so it looks like we do have `$standard-red` that's close (I actually pulled this color off of the 404 page, which is slightly different!), which is defined in `index.scss`, but that doesn't' seem to be referenced outside of that file at all

Reply:
I can see if I can refactor to reference that / maybe put it in a shared place

Reply:
I wouldn't say it's blocking. @nanxiy any reason why we are using two different types of reds here?

Reply:
But yeah in general it seems like all of our links are going to be this color red, so if we can move this out and make all of our hyperlinks this color that would be great